<?xml version="1.0"?>
<launch>
  <!-- Start the differential drive controller -->
  <node pkg="robot_description" type="differential_drive_controller.py" name="diff_drive_controller" output="screen">
    <param name="wheel_radius" value="0.1" />
    <param name="wheel_separation" value="0.42446" />
    <param name="base_frame_id" value="base_link" />
    <param name="odom_frame_id" value="odom" />
  </node>
</launch> 